.github/workflows/ci.yaml

```diff
@@ -18,6 +18,17 @@ jobs:
         uses: actions/checkout@v4
         with:
           fetch-depth: 1
+      - name: Install Buf
+        uses: bufbuild/buf-setup-action@v1
+      - name: Validate Protos
+        uses: bufbuild/buf-breaking-action@v1
+        with:
+          input: proto
+          against: 'https://github.com/connectrpc/conformance.git#branch=v1,subdir=proto' # TODO: Switch to main once merged
```

Comment:
The BSR repo is actually already created: https://buf.build/connectrpc/conformance

Do you think the `breaking` run should be against the BSR? I was thinking that if we have it for PRs, that it should probably be against the Git repo `main` branch right? Since that's where pushes to the BSR will be coming from.

Reply:
If we have a BSR repo, I think we should treat that as the authority. That way, if we ever have a semantic merge conflict, where main is inadvertently made red, a fix can proceed work, since it would be against the BSR -- a last known-good state -- instead of against a possibly-broken head of main.

Reply:
Sounds good.  Updated.